packages/material-ui/src/Typography/Typography.js

```diff
@@ -280,69 +217,43 @@ Typography.propTypes = {
   theme: PropTypes.object.isRequired,
   /**
    * Applies the theme typography styles.
-   * Use `body1` as the default value with the legacy implementation and `body2` with the new one.
    */
-  variant: chainPropTypes(
-    PropTypes.oneOf([
-      'h1',
-      'h2',
-      'h3',
-      'h4',
-      'h5',
-      'h6',
-      'subtitle1',
-      'subtitle2',
-      'body1',
-      'body2',
-      'caption',
-      'button',
-      'overline',
-      'srOnly',
-      'inherit',
-      // deprecated
-      'display4',
-      'display3',
-      'display2',
-      'display1',
-      'headline',
-      'title',
-      'subheading',
-    ]),
-    props => {
-      const deprecatedVariants = [
-        'display4',
-        'display3',
-        'display2',
-        'display1',
-        'headline',
-        'title',
-        'subheading',
-      ];
-      if (
-        props.theme.typography.useNextVariants &&
-        !props.internalDeprecatedVariant &&
-        deprecatedVariants.indexOf(props.variant) !== -1
-      ) {
-        return new Error(
-          'Material-UI: you are using a deprecated typography variant: ' +
-            `\`${props.variant}\` that will be removed in the next major release.\n` +
-            'Please read the migration guide under https://material-ui.com/style/typography#migration-to-typography-v2.',
-        );
-      }
-
-      return null;
-    },
-  ),
+  variant: PropTypes.oneOf([
+    'h1',
+    'h2',
+    'h3',
+    'h4',
+    'h5',
+    'h6',
+    'subtitle1',
+    'subtitle2',
+    'body1',
+    'body2',
+    'caption',
+    'button',
+    'overline',
+    'srOnly',
+    'inherit',
+  ]),
+  /**
+   * We are empirically mapping the variant property to a range of different DOM element types.
+   * For instance, subtitle1 to `<h6>`.
+   * If you wish to change that mapping, you can provide your own.
+   * Alternatively, you can use the `component` property.
+   * The default mapping is the following:
+   */
+  variantMapping: PropTypes.object,
 };
 
 Typography.defaultProps = {
   align: 'inherit',
-  color: 'default',
+  color: 'inherit',
+  display: 'initial',
```

Comment:
should this be `default` instead?

Reply:
Initial is a valid CSS value: https://developer.mozilla.org/en-US/docs/Web/CSS/display. I can't find default.